modules/renter/contractor/contracts.go

```diff
@@ -28,6 +29,15 @@ func (c *Contractor) contractEndHeight() types.BlockHeight {
 	return c.currentPeriod + c.allowance.Period
 }
 
+// ContractUtility returns the ContractUtility for a contract with a given id.
+func (c *Contractor) contractUtility(id types.FileContractID) (modules.ContractUtility, bool) {
```

Comment:
docstring has wrong func name, and I think we actually want this to be `managedContractUtility`